src/pages/ProductListPage/components/ProductSelectButton.tsx

```diff
@@ -0,0 +1,30 @@
+import { ButtonHTMLAttributes, PropsWithChildren } from 'react';
+import addCartIcon from '../../../assets/addCartIcon.png';
+import removeCartIcon from '../../../assets/removeCartIcon.png';
+import styles from '../ProductListPage.module.css';
+
+interface Props extends PropsWithChildren<ButtonHTMLAttributes<HTMLButtonElement>> {
+  isSelected: boolean;
+}
+
+const ProductSelectButton = ({ isSelected, ...props }: Props) => {
+  const buttonStyle = isSelected ? 'remove' : 'add';
+
+  return (
+    <button className={`${styles.productCartButton} ${styles[buttonStyle]}`} {...props}>
+      {isSelected ? (
+        <>
+          <img src={removeCartIcon} width={14} height={14} />
+          빼기
+        </>
+      ) : (
+        <>
+          <img src={addCartIcon} width={14} height={14} />
+          담기
+        </>
+      )}
+    </button>
```

Comment:
```suggestion
    <button className={`${styles.productCartButton} ${styles[buttonStyle]}`} {...props}>
      <img src={isSelected ? removeCartIcon : addCartIcon} width={14} height={14} />
      {isSelected ? '빼기' : '담기'}
    </button>
```
요게 조금 더  간단해보일 거 같은데 어떻게 생각하시나요?